src/common/components/WalletBinding.vue

```diff
@@ -87,86 +114,74 @@ export default {
   }),
 
   computed: {
-    _show: {
-      get() {
-        return this.show;
-      },
-      set(val) {
-        this.$emit("toggle", val);
-      },
-    },
-
     ...mapState({
       api: (state) => state.substrate.api,
       wallet: (state) => state.substrate.wallet,
-      metamaskWalletAddress: (state) => state.metamask.metamaskWalletAddress,
+      metamaskWalletAddress: (state) => state.metamask.metamaskWalletAddress
     }),
-  },
 
-  watch: {
-    show() {
-      if (this.show) {
-        this.error = "";
-        this.loading = false;
-        this.putAccount = false;
-        this.putWallet = true;
-        this.inputPassword = false;
-        this.isLoading = false;
-      }
-    },
+    watch: {
+      show() {
+        if (this.show) {
+          this.error = "";
+          this.loading = false;
+          this.putAccount = false;
+          this.putWallet = true;
+          this.inputPassword = false;
+          this.isLoading = false;
```

Comment:
Please avoid redundant statements like this, should we prevent with `if (!this.show) return`